causalml/inference/meta/rlearner.py

```diff
@@ -307,8 +307,8 @@ def __init__(self,
             n_fold=n_fold,
             random_state=random_state)
 
-        if (outcome_learner is None) and (effect_learner is None):
-            raise ValueError("Either the outcome learner or the effect learner must be specified.")
+        # if (outcome_learner is None) and (effect_learner is None):
+        #    raise ValueError("Either the outcome learner or the effect learner must be specified.")
```

Comment:
Note that the assert should be different for the classifiers because meta-regressors can use the same learner for both "outcome" and "effect" while the meta-classifiers can't.

Reply:
I see. In that case, let's check if the outcome learner has the `predict_proba` method. @lawinslow, could you add it for the classifiers?

Reply:
Yeah, good point. I actually hadn't fully realized that before. Will fix.